layouts/shortcodes/thirdparty-content.html

```diff
@@ -0,0 +1,4 @@
+<blockquote class="callout caution" role="alert">
```

Comment:
We could do:
```suggestion
<blockquote class="callout caution thirdpartycontent" role="alert">
```
Anyway, good to get this done and to iterate. The perfect is the enemy of the merged.